duckduckgo/Android
# Remove All Fireproof Sites Feature

Task/Issue URL: [https://app.asana.com/0/72649045549333/1201953975647701/f](https://app.asana.com/0/72649045549333/1201953975647701/f)

### Description
New toolbar menu option for removing all fireproof sites

### Steps to test this PR

_Remove All Fireproof sites_
- Install from this branch
- Go to Settings > Fireproof Sites and make sure you have 1 or more fireproof sites added
- Tap on toolbar overflow menu > Remove All
- [x] Check all Fireproof sites are removed
- [x] Check empty state text appears

_Undo Remove All Fireproof sites_
- Do all previous steps from _Remove all Fireproof sites_
- [x] Check Snackbar shows with `undo` option
- Tap on "undo" option
- [x] Check all Fireproof sites are back in the list

### UI changes
| Before  | After |
| ------ | ----- |
![Screenshot_20220427-204034_DuckDuckGo](https://user-images.githubusercontent.com/20798495/165597367-c5fa9406-b44f-4c97-9cc8-f7c8009a6be5.jpg) | ![Screenshot_20220427_203855](https://user-images.githubusercontent.com/20798495/165597221-cfe8bcc4-7b1c-4f43-a5b4-3835ace99e4c.png)
|
